app-security/clamav/clamav-1.2.0.recipe

```diff
@@ -0,0 +1,125 @@
+DESCRIPTION="ClamAV is an open source antivirus engine for detecting  \
+trojans, viruses, malware & other malicious threats."
+HOMEPAGE="https://www.clamav.net"
+COPYRIGHT="2001-2007 Tomasz Kojm
+	2007-2023 Cisco Systems"
+PACKAGER="Luca D'Amico <damico.luca91@live.it>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/Cisco-Talos/clamav/archive/refs/tags/clamav-$portVersion.tar.gz"
+CHECKSUM_SHA256="efe7e234fe29a96bf4da23336e38ea2989ed0be162342993e99a11901524f43f"
+SOURCE_DIR="clamav-clamav-$portVersion"
+PATCHES="haiku-$portVersion.patch"
+ADDITIONAL_FILES="
+	clamd.conf
+	freshclam.conf
+	"
+
+ARCHITECTURES="all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	clamav$secondaryArchSuffix = $portVersion
+	cmd:clambc$secondaryArchSuffix = $portVersion
+	cmd:clamconf$secondaryArchSuffix = $portVersion
+	cmd:clamd$secondaryArchSuffix = $portVersion
+	cmd:clamdscan$secondaryArchSuffix = $portVersion
+	cmd:clamdtop$secondaryArchSuffix = $portVersion
+	cmd:clamscan$secondaryArchSuffix = $portVersion
+	cmd:clamsubmit$secondaryArchSuffix = $portVersion
+	cmd:freshclam$secondaryArchSuffix = $portVersion
+	cmd:sigtool$secondaryArchSuffix = $portVersion
+	lib:libclamav$secondaryArchSuffix = $portVersion
+	lib:libclammspack$secondaryArchSuffix = $portVersion
+	lib:libclamunrar$secondaryArchSuffix = $portVersion
+	lib:libclamunrar_iface$secondaryArchSuffix = $portVersion
+	lib:libfreshclam$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	json_c$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libpcre2_8$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	check${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libpcre2_8$secondaryArchSuffix
+	devel:libpthread_stubs$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	json_c${secondaryArchSuffix}_devel
+	python3.12$secondaryArchSuffix
+	rust_bin$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+GLOBAL_WRITABLE_FILES="
+	settings/clamav/clamd.conf keep-old
+	settings/clamav/freshclam.conf keep-old
+	settings/clamav/db directory keep-old
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. -DENABLE_MILTER=OFF \
+	-DAPP_CONFIG_DIRECTORY=$settingsDir/clamav \
+	-DDATABASE_DIRECTORY=$settingsDir/clamav/db \
+	-DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_BUILD_TYPE=Release .
+
+	make -C build
+}
+
+INSTALL()
+{
+	mkdir -p $binDir $libDir $manDir/man1 $manDir/man5 $manDir/man8
```

Comment:
There is a install part here, maybe use that instead of cp'n manualy? (also installs headers, so maybe add a devel package?)
```
	cmake -B build install
```